terminus-ui/src/select/select.component.ts

```diff
@@ -31,6 +31,7 @@ import {
   isString,
   TsDocumentService,
   untilComponentDestroyed,
+  isFunction,
```

Comment:
We've started keeping our imports somewhat alphabetical. Currently we are alphabetizing import collections like the above, but not every separate import statement.

This may make it easier: https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines